### PR TITLE
Riverne portals

### PR DESCRIPTION
**Pull request by DFPercush**
Client update changed event/npc IDs in Riverne A01 and B01 making it difficult to navigate the zone and complete CoP missions. Remapped and tested all portals both ways in each zone. One unstable displacement in each zone continues to be uncoded, but they appear to be dead ends anyway.

### CODE REVIEW

File: scripts/zones/Riverne-Site_A01/npcs/Spatial_Displacement.lua [hunk 1]
```diff
@@ -17,67 +17,11 @@ function onTrigger(player,npc)
...
-    elseif (id == base+38) then
-        player:startEvent(26);
-    elseif (id == base+39) then
-        player:startEvent(27);
-    end
+    elseif (id - base <= 39) then
+        player:startEvent(id - base);
+    end;
+    
+end;
```
Comment (Wiggo32) [inline scripts/zones/Riverne-Site_A01/npcs/Spatial_Displacement.lua]: Except for this random end; sitting here... this script works. All of the displacements teleported me to the proper location. (Haven't tested B01 yet)

**Comment by wrenffxi** [comment]
am working these fixes into the ID cleanup for riverne zones.

**Comment by takhlaq** [comment]
@wrenffxi could you pull his changes into your branch and correct the syntax error, that way his original commit is still in the commit log

File: scripts/zones/Riverne-Site_A01/npcs/Spatial_Displacement.lua (hunk 1 above)
Comment (TeoTwawki) [reply scripts/zones/Riverne-Site_A01/npcs/Spatial_Displacement.lua]: Looks like the end belongs there, but it got a semicolon added so git sees it as a new one.

**Comment by wrenffxi** [comment]
Shoot, I didn't see this until after I made separate PR.  I didn't actually use the code from this PR's commit, only the portal id / event id pairs.  (I re-checked them all, and can confirm accuracy)

**Comment by takhlaq** [comment]
https://github.com/DarkstarProject/darkstar/pull/4544

**Pull request closed without merging**